runtime/server_http_request.go

```diff
@@ -651,6 +651,20 @@ func (req *ServerHTTPRequest) UnmarshalBody(
 	return true
 }
 
+// WriteBytesBody replace the request body
+// we only change things cached in this layer, not in the raw httpRequest
+// Assumption is that encoding does not change
+func (req *ServerHTTPRequest) WriteBytesBody(
```

Comment:
`ReplaceBodyWith` is probably a more descriptive name,  also the line breaking is unnecessary.

Reply:
agree